modules/serverless-minimal-requirements-serving.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="serverless-minimal-requirements-serving_{context}"]
+= Minimal requirements of OpenShift Serverless Serving
+
+While the default setup is suitable for medium-sized workloads, it might be over-sized for smaller setups or under-sized for high-workload scenarios.
+To configure OpenShift Serverless Serving for a minimal workload scenario, you need to know the idle consumption of the system components.
+
+[id="serverless-minimal-requirements-serving-idle-consumption_{context}"]
+== Idle consumption
+
+The idle consumption is dependent on the number of Knative Services. The following memory usage has been measured for the components in the `knative-serving` and `knative-serving-ingress` OpenShift projects:
+
+[cols=5*,options="header"]
+|===
+|Component
```

Comment:
The component names are in backticks usually, probably should be in this first column as well?